script.py

```diff
@@ -8,12 +8,12 @@
 
 def main():
     if token:
-        response = request_vehicles(token)
+        response = request_vehicles(None)
         print_info(response)
-        return 0
+        sys.exit(signal)
```

Comment:
Will `signal` ever be changed? If not, you can just use `sys.exit()` to exit the program with the status code 0.